src/clarity-angular/popover/signpost/_signposts.clarity.scss

```diff
@@ -0,0 +1,389 @@
+.signpost {
+    display: inline;
+
+    &:hover {
+        cursor: pointer;
+    }
+
+    button {
+        min-width: $clr_baselineRem_1;
+        margin: 0 $clr_baselineRem_0_3;
+        padding: 0;
+        color: $clr-dark-midtone-gray!important;
+        outline: none;
+
+        clr-icon {
+            width: $clr_baselineRem_1;
+            height: $clr_baselineRem_1;
+        }
+
+        &:hover, &.active {
+            color: $clr-action-blue!important;
+        }
+    }
+
+    /***
+        The signpost is above the icon top-{HORIZONTAL_POSITION}
+    */
+    &.top-left {
+        .signpost-content {
+            .popover-pointer {
+                width: 0;
```

Comment:
can extract the `width` and `height` which remain `0` throughout & also the `position: absolute` right?